zcncore/get_data.go

```diff
@@ -177,7 +181,7 @@ func GetMinerSCNodeInfo(id string) ([]byte, error) {
 		return nil, err
 	}
 
-	return client.MakeSCRestAPICall(MinerSmartContractAddress, GET_MINERSC_NODE, Params{
+	return node.MakeSCRestAPICallToSharder(MinerSmartContractAddress, GET_MINERSC_NODE, Params{
```

Comment:
Can we not use 0box calls here? If not why?

Reply:
This is not used by any sdk (wasm or mobile).

Reply:
https://github.com/0chain/gosdk/pull/1685#discussion_r1855204990